### ISSUE TITLE
How about releasing it as an official idea plugin?

### ISSUE THREAD

**Issue by fno2010** [snlab-freedom/Jetbrains-YANG#1]
@LovingYoung Is it possible to distribute it into the official idea plugin repo?

**Comment by yjlcoder** [comment]
Yeah, of course. Maybe we could publish it as Alpha0.1 when we make some progress.

**Comment by fno2010** [comment]
Before alpha release, a major recommendation from me is to introduce a project build system (maven or gradle) and use some CI (maybe travis.ci) for the automatic building.

**Comment by fno2010** [comment]
@LovingYoung I just linked this repo to [travis-ci](https://travis-ci.org/snlab-freedom/Jetbrains-YANG). Next time you want to publish a release to GitHub, you can just commit a tag. :)

**Comment by fno2010** [comment]
[This project](https://github.com/intellij-rust/intellij-rust) seems to be a better reference.
